components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -1736,6 +1737,146 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
             .map((s) => s.url);
     }
 
+    public async getSuggestedRepositories(ctx: TraceContext, organizationId: string): Promise<SuggestedRepository[]> {
+        const user = await this.checkAndBlockUser("getSuggestedRepositories");
+        await this.guardWithFeatureFlag("includeProjectsOnCreateWorkspace", user, organizationId);
+
+        if (!uuidValidate(organizationId)) {
+            throw new ApplicationError(ErrorCodes.BAD_REQUEST, "organizationId must be a valid UUID");
+        }
+
+        const logCtx: LogContext = { userId: user.id };
+
+        type SuggestedRepositoryWithSorting = SuggestedRepository & {
+            priority: number;
+            lastUse?: string;
+        };
+
+        const fetchExampleRepos = async (): Promise<SuggestedRepositoryWithSorting[]> => {
```

Comment:
Let's not add the example repos anymore. I don't think they are helpful if we can show stuff that belongs to the user. cc @loujaybee

Reply:
Thanks for highlighting, Sven! I'm not fully sure what removing the demo repos would mean, or the consequence? Where currently in the product do we surface the example repos today? What would be the impact/change? 

> if we can show stuff that belongs to the user.

If I understand correctly, I agree. I'd say for most of our customers we should work on the assumption that there will be a central team that wants to actively manage and curate the experience, and that could include making projects that are examples or demos for their teams. That could be something that we could bootstrap our organisations with (a couple example real projects) as part of customer onboarding.

Reply:
@loujaybee In this case it would exclude example repos in what shows up in the repo selector on the create workspace page.

Reply:
Re-posting two of the current problems with the **example repositories**:
1. Users who haven't authorized GitHub.com, see none of them in `/new`, including users using GitHub Enterprise, see #3531 and [relevant discussion](https://gitpod.slack.com/archives/C01KPEPLLRY/p1674207062231199?thread_ts=1674158268.615469&cid=C01KPEPLLRY) (internal). :x:
2. From those users who can see them in `/new`, they don't have permissions to push (write access) for these repositories, making them a bit unusable as they have to fork or add a git remote before pushing any changes. :x: